src/app/modules/startup/internal/keycard_create_pin_state.nim

```diff
@@ -13,11 +13,15 @@ proc delete*(self: KeycardCreatePinState) =
 method executeBackCommand*(self: KeycardCreatePinState, controller: Controller) =
   controller.setPin("")
   controller.setPinMatch(false)
+  if self.flowType == FlowType.FirstRunNewUserNewKeycardKeys or
```

Comment:
this check appears several times. Can it be generalised with static list: if KEYCARD_FLOW_TYPES.find(self.flowType)